lib/honeybadger/plugins/rails.rb

```diff
@@ -33,6 +33,22 @@ def render_exception(arg, exception)
         requirement { defined?(::Rails.application) && ::Rails.application }
 
         execution do
+          if ::Rails.application.config.exceptions_app.is_a?(::ActionDispatch::Routing::RouteSet)
+            Honeybadger.configure do |config|
```

Comment:
I think you could remove this outer configure block because `config` is available in `execution` directly. I don't really have a strong opinion about it.

Reply:
this is interesting... because if I use the `config` directly (yeah, this is an instance of `Honeybadger::Config`), the hook isn't executed... 🤔 if it's a deal, needs more investigation about the hooks execution, what do you think?

Reply:
Huh, that's strange. Well, I'm OK with keeping `Honeybadger.config` for now. I can look into the issue of why `config` doesn't work for configuring hooks separately. Thanks for checking on it. :)

Reply:
ok, thank you 😄